File: Frontend/src/app/models/student/studentChangeDto.ts
import {Guid} from "guid-typescript";

export interface StudentChangeDto
{
  id:Guid,
  fullName:string,
  birthDate: Date,
  universityName:string,
  course: number,
  faculty:string,
  phoneNumber:string,
  email:string,
  description:string,
  changeUser:string
}

File: Backend/Lenta.Api/Dto/Student/StudentChangeDto.cs
namespace Lenta.Api.Dto.Student;

public class StudentChangeDto
{
    public Guid Id { get; set; }
    public string FullName { get; set; } = null!;
    public string Email { get; set; } = null!;
    public DateTimeOffset BirthDate { get; set; }
    public string UniversityName { get; set; } = null!;
    public int Course { get; set; }
    public string Faculty { get; set; } = null!;
    public string PhoneNumber { get; set; } = null!;
    public string Description { get; set; } = null!;
    public string ChangeUser { get; set; } = null!;
}